src/routes/jira/server/jira-server-url-post.ts

```diff
@@ -12,15 +12,18 @@ interface GheServerUrlErrors {
 	[key: string | number]: MessageAndCode;
 }
 
+const GHE_ERROR_ENOTFOUND = {
+	errorCode: "GHE_ERROR_ENOTFOUND",
+	message: "Request to URL failed"
+};
+
 export const gheServerUrlErrors: GheServerUrlErrors = {
 	invalidUrl: {
 		errorCode: "GHE_ERROR_INVALID_URL",
 		message: "Invalid URL"
 	},
-	ENOTFOUND: {
-		errorCode: "GHE_ERROR_ENOTFOUND",
-		message: "Request to URL failed"
-	},
+	ENOTFOUND: GHE_ERROR_ENOTFOUND,
+	403: GHE_ERROR_ENOTFOUND,
```

Comment:
I worry it may confuse other devs as `GHE_ERROR_ENOTFOUND` looks like `404 NOT FOUND`, but here what we are trying to say is "I can't reach the GHE server url you provided." 
So maybe `GHE_ERROR_UNREACHABLE`?